securedrop/manage.py

```diff
@@ -167,11 +169,45 @@ def add_admin():
             break
 
     while True:
+        print "Please enter a password.\n\nThe password must be at least 8 characters and contain a combination of upper\nand lower characters, numbers and special characters.\n"
```

Comment:
We try to avoid having lines that are > 80 characters long (see PEP8). There are several ways to fix this in Python, but here I would recommend using a multiline string because the string contains newlines.

```py
print """Please enter a password.

The password must be at least 8 characters and contain a combination of upper
and lower characters, numbers and special characters.
"""